### PR TITLE
Fix agent iso upload to vSphere datastore

### PR DESCRIPTION
**Pull request by mhanss**
[OCPBUGS-15775](https://issues.redhat.com/browse/OCPBUGS-15775)

### CODE REVIEW

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-agent-sno-f14

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-agent-sno-f14

**Comment by mhanss** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-agent-sno-f14

File: ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh [hunk 1]
```diff
@@ -236,12 +236,22 @@ agent_iso=$(<"${SHARED_DIR}"/agent-iso.txt)
 echo "uploading ${agent_iso} to iso-datastore.."
 
 for ((i = 0; i < 3; i++)); do
-  if /tmp/govc datastore.upload -ds "${vsphere_datastore}" agent.x86_64.iso agent-installer-isos/"${agent_iso}"; then
-    echo "$(date -u --rfc-3339=seconds) - Agent ISO has been uploaded successfully!!"
-    status=0
-    break
+  file_upload_url="https://${vsphere_url}/folder/agent-installer-isos/${agent_iso}?dcPath=${vsphere_datacenter}&dsName=${vsphere_datastore}"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh:239]: Out of curiousity, how is this different from govc?

**Pull request closed without merging**